packages/kununu-header/index.jsx

```diff
@@ -0,0 +1,9 @@
+import Header from './Header';
+import HeaderNav from './HeaderNav';
+import HeaderNavItem from './HeaderNavItem';
+
+Header.Header = Header;
+Header.HeaderNav = HeaderNav;
+Header.HeaderNavItem = HeaderNavItem;
+
+module.exports = Header;
```

Comment:
Can all of this be done with [the `export {} from` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)?

```js
export {Header} from './Header';
export {HeaderNav} from './HeaderNav';
export {HeaderNavItem} from './HeaderNavItem';
```